Panic when multiple string values passed into Member function.

When attempting to generate code from this design. The goagen binary panics on an `invalid memory address or nil pointer deference`, and I have tracked it down to the multiple string values passed into the Member function.

This was produced using the latest `v1` code.

Code
```go
package replication

import (
	. "github.com/goadesign/goa/design"
	. "github.com/goadesign/goa/design/apidsl"
)

var _ = Resource("Test", func() {
	Action("create", func() {
		Routing(POST("//lists/:listID/tasks"))
		Payload(func() {
			Member("id", "value")
		})
		Response(OK)
	})
})
```

Result:
```
./vendor/github.com/goadesign/goa/goagen/goagen --design=github.com/rocketmade/goa-todo-api/replication bootstrap
exit status 2
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4b91b5]

goroutine 1 [running]:
panic(0x796900, 0xc4200160e0)
    /usr/lib/go/src/runtime/panic.go:500 +0x1a1
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*MediaTypeDefinition).IsPrimitive(0x0, 0xc4201b68e8)
    <autogenerated>:115 +0x5
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*AttributeDefinition).Validate(0xc4201d7600, 0xc4201b6930, 0xb, 0x9f6fe0, 0xc4201bc500, 0xc4201b68e8)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/validation.go:451 +0x139
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*AttributeDefinition).Validate(0xc4201d7580, 0xc4201bc560, 0x11, 0x9f6fe0, 0xc4201bc500, 0xc4201baea0)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/validation.go:479 +0x451
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*UserTypeDefinition).Validate(0xc4201bc500, 0x80fad0, 0xe, 0x9f6d20, 0xc4201ea280, 0x0)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/validation.go:520 +0xa8
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*ActionDefinition).Validate(0xc4201ea280, 0xc4201bac60)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/validation.go:333 +0x886
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*ResourceDefinition).validateActions(0xc4201f8000, 0xc4201bae40)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/validation.go:230 +0xe2
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*ResourceDefinition).Validate(0xc4201f8000, 0xc4201bc540)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/validation.go:208 +0x6f
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*APIDefinition).Validate.func1(0xc4201f8000, 0xc4201bac30, 0x7fb796)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/validation.go:90 +0x55
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*APIDefinition).IterateResources(0xc4201f4000, 0xc42022ba80, 0xc42022bb40, 0x8)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/definitions.go:607 +0x1d8
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*APIDefinition).Validate(0xc4201f4000, 0x9f6ce0, 0xc4201f4000)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/validation.go:117 +0x232
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/dslengine.validateSet(0xc4201b6900, 0x1, 0x1, 0xa36940, 0x0)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/dslengine/runner.go:312 +0x120
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design.(*APIDefinition).IterateSets(0xc4201f4000, 0x84de08)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/design/definitions.go:458 +0x96
github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/dslengine.Run(0x9f8460, 0xc4201bc340)
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/vendor/github.com/goadesign/goa/dslengine/runner.go:108 +0x197
main.main()
    /home/zettazete/go/src/github.com/rocketmade/goa-todo-api/goagen397620282/main.go:27 +0x63
```

--- a/design/apidsl/attribute.go
+++ b/design/apidsl/attribute.go
@@ -155,8 +155,12 @@ func parseAttributeArgs(baseAttr *design.AttributeDefinition, args ...interface{
 		if name, ok2 := args[index].(string); ok2 {
 			// Lookup type by name
 			if dataType, ok = design.Design.Types[name]; !ok {
-				if dataType = design.Design.MediaTypeWithIdentifier(name); dataType == nil {
+				var mt *design.MediaTypeDefinition
+				if mt = design.Design.MediaTypeWithIdentifier(name); mt == nil {
+					dataType = design.String // not nil to avoid panics
 					dslengine.InvalidArgError(expected, args[index])
+				} else {
+					dataType = mt
 				}
 			}
 			return

